yarn-project/simulator/src/public/public_processor.ts

```diff
@@ -91,9 +95,11 @@ export class PublicProcessor {
     protected historicalHeader: Header,
     protected publicContractsDB: ContractsDataSourcePublicDB,
     protected publicStateDB: PublicStateDB,
-
+    telemetryClient: TelemetryClient,
     private log = createDebugLogger('aztec:sequencer:public-processor'),
-  ) {}
+  ) {
+    this.tracer = telemetryClient.getTracer('@aztec/simulator:PublicProcessor');
```

Comment:
Other `getTracer` calls don't include the package name. Why do we include it here?

Reply:
I've not been too strict with the tracer names on this PR. In the [spec](AztecProtocol/engineering-designs#1) we said we'd use class names. I'll update this to `PublicProcessor`.